atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -306,9 +306,13 @@ private boolean trySchemaMutationUnlockOnce(long perOperationNodeId) {
         try {
             result = clientPool.runWithRetry(client -> {
                 Column existingColumn = queryExistingLockColumn(client);
+                long existingLockId = getLockIdFromColumn(existingColumn);
 
-                Preconditions.checkState(isValidColumnForLockId(existingColumn, perOperationNodeId),
-                        "Trying to unlock unowned lock");
+                if (existingLockId != GLOBAL_DDL_LOCK_CLEARED_ID) {
```

Comment:
Sure, we could update this so that runWithRetry returns a boolean. Not sure how we will test this though, since I believe we don't have a way of just trying an unlock. The only API we have is `runWithLock` right?